targets/TARGET_STM/TARGET_STM32L0/device/stm32l0xx_hal_spi.c

```diff
@@ -1009,7 +1024,16 @@ HAL_StatusTypeDef HAL_SPI_Transmit_IT(SPI_HandleTypeDef *hspi, uint8_t *pData, u
     hspi->State        = HAL_SPI_STATE_BUSY_TX;
     hspi->ErrorCode    = HAL_SPI_ERROR_NONE;
 
-    hspi->TxISR        = &SPI_TxISR;
+    /* Set the function for IT treatment */
+    if(hspi->Init.DataSize > SPI_DATASIZE_8BIT )
+    {
+        hspi->TxISR = SPI_TxISR_16BIT;
```

Comment:
Indentation here is inconsistent with the rest of the function which uses 2 spaces